### ISSUE TITLE
deletion of a group is not allowed without a policy set on it

### ISSUE THREAD

**Issue by cesnietor** [minio/minio#9262]
While creating a group with mc admin and then setting the group empty, `minio` is not allowing me to delete the group unless it has a policy set on it
This scenario is considering that the group is already empty with no users in it.
## Expected Behavior
I should be able to delete the group as admin without having to set a policy on the group. 

## Current Behavior
Trying to delete the group 
```
./mc admin group remove myminio group1          
mc: <ERROR> Could not perform remove operation. The canned policy does not exist.
```
I can only delete a group after setting a policy on it.
```
./mc admin policy set myminio readwrite group=group1
Policy readwrite is set on group `group1`
./mc admin group remove myminio group1              
Removed group group1 successfully.
```
## Possible Solution
not requiring a policy on the group to be deleted

## Steps to Reproduce (for bugs)

1. Create a user
2. Create a group with the user created on step 1
3. Remove the user from the group so that now it is empty
4. Delete the empty group

## Your Environment

* Version used (`minio version`): `RELEASE.2020-04-02T21-34-49Z`
* Environment name and version (e.g. nginx 1.9.1):
* Server type and version:
* Operating System and version (`uname -a`):
* Link to your project:

**Comment by donatello** [comment]
@cesnietor This can be closed now?

**Comment by cesnietor** [comment]
> @cesnietor This can be closed now?

yes thank you! Solved on #9288
